cmd/maya-apiserver/cstor-operator/cspc/operations.go

```diff
@@ -108,22 +121,31 @@ func getRaidGroupType(group apis.RaidGroup, poolSpec *apis.PoolSpec) string {
 }
 
 // addBlockDeviceToGroup adds block devices to the provided raid group on CSP
-func addBlockDeviceToGroup(group *apis.RaidGroup, csp *apis.NewTestCStorPool) *apis.NewTestCStorPool {
+func (pc *PoolConfig) addBlockDeviceToGroup(group *apis.RaidGroup, csp *apis.NewTestCStorPool) *apis.NewTestCStorPool {
 	for i, groupOnCSP := range csp.Spec.RaidGroups {
 		groupOnCSP := groupOnCSP
 		if isRaidGroupPresentOnCSP(group, csp) {
 			if len(group.BlockDevices) > len(groupOnCSP.BlockDevices) {
 				newBDs, err := getAddedBlockDevicesInGroup(group, &groupOnCSP)
```

Comment:
`error` return value not required for getAddedBlockDevicesInGroup..
let us remove check in that `getAddedBlockDevicesInGroup` for `len(addedBlockDevice) == 0`

Reply:
its unnecessary code

Reply:
accordingly, next `err != null` also can be removed

Reply:
Yes. Good catch. thanks